src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[] {
+                new Truck(),
+                new Bulldozer(),
+                new Excavator()
```

Comment:
It's good practice to use abstract references instead of specific ones where possible. This allows for more flexible code and the ability to use polymorphism. Consider declaring the array with the type of the abstract class 'Machine' instead of specific machines.